dusty/commands/test.py

```diff
@@ -143,10 +143,8 @@ def _run_tests_with_image(client, expanded_specs, app_or_lib_name, test_command,
 
     for service_container in previous_container_names:
         log_to_client('Stopping service container {}'.format(service_container))
-        try:
-            client.stop(service_container, timeout=1)
-        except Exception as e:
-            log_to_client('Exception stopping service container {}: {}'.format(service_container, str(e)))
+        client.kill(service_container)
```

Comment:
Why do we care if they die with dignity if we are just going to remove all traces of this container next.   Dieing with dignity is reserver for containers we respect. We do not care or respect this container.